decentralize-rbac-app/src/endpoints/authz.ts

```diff
@@ -0,0 +1,39 @@
+import * as ccfapp from "@microsoft/ccf-app";
+import { ApiResult, CCFResponse } from "../utils/api-result";
+import authenticationService from "../services/authentication-service";
+import authzService from "../services/authz-service";
+import { ServiceResult } from "../utils/service-result";
+import { Service } from "protobufjs";
+
+/**
+ * HTTP GET Handler for checking if a user exists
+ * @param {ccfapp.Request<any>} request - mTLS request with userId and CSV file for ingestion
+ * @returns {ServiceResult<string>} - data has been ingested successfully
+ */
+export function authorize(request: ccfapp.Request<any>): ccfapp.Response<CCFResponse> {
+  // check if caller has a valid identity
+  const isValidIdentity = authenticationService.isAuthenticated(request);
+  if (isValidIdentity.failure)
+    return ApiResult.AuthFailure();
+
+  const userId = request.params.user_id;
+  const action = request.params.actionName;
+
+  if (!userId || !action){
+    return ApiResult.Failed(ServiceResult.Failed({
+      errorMessage: "UserId and action are required",
```

Comment:
```suggestion
      errorMessage: "userId and action are required",
```

Seems clearer to keep the same capitalisation as in the message to avoid user confusion.